src/404.html

```diff
@@ -0,0 +1,61 @@
+---
+title: "We couldn't find that Web page (Error 404) / <span lang='fr'>Nous ne pouvons trouver cette page Web (Erreur 404)</span>"
+permalink: 404.html
+---
+<!DOCTYPE html>
+<html class="no-js" lang="en" dir="ltr">
+
+	<head>
+		<meta charset="utf-8" />
+		<!-- Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+		wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html -->
+		<title>{{ title | striptags(true) | escape | nl2br }}</title>
+		<meta content="width=device-width, initial-scale=1" name="viewport" />
+		<meta name="robots" content="noindex, nofollow, noarchive" />
+		<link href="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/favicon.ico" rel="icon" type="image/x-icon" />
+		<link rel="stylesheet" href="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/css/theme-srv.css" />
+
+	</head>
+
+	<body vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
+		<header id="wb-bnr" class="container">
+			<div class="row" property="publisher" resource="#wb-publisher" typeof="GovernmentOrganization">
+				<div class="col-sm-6">
+					<img id="gcwu-sig" src="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/sig-blk-en.svg" alt="Government of Canada" property="logo" /><span class="wb-inv"> / <span lang="fr">Gouvernement du Canada</span></span>
+					<meta property="name" content="Government of Canada" />
+					<meta property="areaServed" typeOf="Country" content="Canada" />
+				</div>
+				<div class="col-sm-6">
+					<img id="wmms" src="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/wmms-blk.svg" alt="Symbol of the Government of Canada" property="logo" /><span class="wb-inv"> / <span lang="fr">Government of Canada</span></span>
+				</div>
+			</div>
+		</header>
+		<main property="mainContentOfPage" resource="#wb-main" typeof="WebPageElement" class="container">
+			<div class="row mrgn-tp-lg">
+				<h1 class="wb-inv">{{ title }}</span></h1>
+
+				<section class="col-md-6">
+					<h2><span class="glyphicon glyphicon-warning-sign mrgn-rght-md"></span> We couldn't find that Web page (Error 404)</h2>
+					<p>We're sorry you ended up here. Sometimes a page gets moved or deleted, but hopefully we can help you find what you're looking for.</p>
+					<ul>
+						<li>Return to the <a href="/en/">home page</a></li>
+					</ul>
+				</section>
+
+				<section class="col-md-6" lang="fr">
+					<h2><span class="glyphicon glyphicon-warning-sign mrgn-rght-md"></span> Nous ne pouvons trouver cette page Web (Erreur 404)</h2>
+					<p>Nous sommes désolés que vous ayez abouti ici. Il arrive parfois qu'une page ait été déplacée ou supprimée. Heureusement, nous pouvons vous aider à trouver ce que vous cherchez.</p>
```

Comment:
"abouti" ne me semble être le mot idéal.

"Hopefully" devrait être remplacé. Dans ce contexte "Heureusement" ne fonctionne pas bien.

```suggestion
					<p>Nous sommes désolés que vous avez arrivé sur cette page. Il arrive parfois qu'une page ait été déplacée ou supprimée. Nous espérons que les liens suivants vont vous permettre de trouver ce que vous cherchez.</p>
```